content/en/storage-providers/setup/initialize.md

```diff
@@ -60,6 +60,7 @@ Before you run your storage provider, it is important that it is reachable from
 
 Once you start your storage provider, [make sure you can connect to its public IP/port]({{< relref "../../storage-providers/operate/connectivity/" >}}).
 
+
```

Comment:
Seems like there is still an extra line here. Not sure if it would cause any conflicts with exciting commits in that page. But might be ideal to leave out any change on that page.

Reply:
All the merge conflicts have already been resolved with last commit.